src/test/resources/data/request_jsons/mocked_request1_complete_tumor_normal.json

```diff
@@ -0,0 +1,235 @@
+              "runDate": "2018-06-05",
+              "flowCellLanes": [
+                5
+              ],
+              "fastqs": [
+                "/FASTQ/Project_MOCKREQUEST1_B/Sample_01-0012345a_IGO_MOCKREQUEST1_B_2/01-0012345a_IGO_MOCKREQUEST1_B_2_S83_R2_001.fastq.gz",
+                "/FASTQ/Project_MOCKREQUEST1_B/Sample_01-0012345a_IGO_MOCKREQUEST1_B_2/01-0012345a_IGO_MOCKREQUEST1_B_2_S83_R1_001.fastq.gz"
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "igoId": "MOCKREQUEST1_B_3",
+      "cmoSampleName": "C-8DH24X-X001-d",
+      "sampleName": "XXX002_P1_12348a_R_43",
+      "cmoSampleClass": "Primary",
+      "cmoPatientId": "C-8DH24X",
+      "investigatorSampleId": "XXX002_P1_12348a_R_43",
+      "tumorOrNormal": "Tumor",
+      "tissueLocation": "",
+      "specimenType": "PDX",
+      "sampleOrigin": "Tissue",
+      "preservation": "Frozen",
+      "collectionYear": "",
+      "sex": "M",
+      "species": "Human",
+      "tubeId": "",
+      "baitSet": "GENESET101_BAITS",
+      "qcReports": [],
+      "cmoSampleIdFields": {
+  		"naToExtract" :"testingNaToExtractvalue",
+  		"normalizedPatientId": "testingNormalizedPatientIdvalue"
+		},
+      "libraries": [
+        {
+          "barcodeId": "TNG53",
+          "barcodeIndex": "GCTCGGTA",
+          "libraryIgoId": "MOCKREQUEST1_B_3_1_1_1",
+          "libraryVolume": 35.0,
+          "libraryConcentrationNgul": 35.2,
+          "captureConcentrationNm": "11.363636363636363",
+          "captureInputNg": "400.0",
+          "captureName": "Pool-MOCKREQUEST1_B-Tube7_1",
+          "runs": [
+            {
+              "runMode": "HiSeq High Output",
+              "runId": "RUNID_0123",
+              "flowCellId": "X5KL2KKAY",
+              "readLength": "",
+              "runDate": "2018-06-05",
+              "flowCellLanes": [
+                5
+              ],
+              "fastqs": [
+                "/FASTQ/Project_MOCKREQUEST1_B/Sample_XXX002_P1_12348a_R_43_IGO_MOCKREQUEST1_B_3/XXX002_P1_12348a_R_43_IGO_MOCKREQUEST1_B_3_S84_R1_001.fastq.gz",
+                "/FASTQ/Project_MOCKREQUEST1_B/Sample_XXX002_P1_12348a_R_43_IGO_MOCKREQUEST1_B_3/XXX002_P1_12348a_R_43_IGO_MOCKREQUEST1_B_3_S84_R2_001.fastq.gz"
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "igoId": "MOCKREQUEST1_B_4",
+      "cmoSampleName": "C-8DH24X-N001-d",
+      "sampleName": "01-XXXXXXXa",
+      "cmoSampleClass": "Normal",
+      "cmoPatientId": "C-8DH24X",
+      "investigatorSampleId": "01-XXXXXXXa",
+      "tumorOrNormal": "Normal",
+      "tissueLocation": "",
+      "specimenType": "Blood",
+      "sampleOrigin": "Whole Blood",
+      "preservation": "Frozen",
+      "collectionYear": "",
+      "sex": "M",
+      "species": "Human",
+      "tubeId": "",
+      "baitSet": "GENESET101_BAITS",
+      "qcReports": [],
+      "cmoSampleIdFields": {
+  		"naToExtract" :"testingNaToExtractvalue",
```

Comment:
same comment here - i would confirm that the spacing is consistent everywhere. You might be able to just copy and paste this whole json into something like http://jsonprettyprint.net/ to prettify it and then just replace the file contents with the output from the program